samples/cpp/facedetect.cpp

```diff
@@ -122,36 +114,27 @@ int main( int argc, const char** argv )
         if(image.empty()) cout << "Couldn't read ../data/lena.jpg" << endl;
     }
 
-    cvNamedWindow( "result", 1 );
+    namedWindow( "result", 1 );
```

Comment:
Well the only reason why one would create a `namedWindow` is if he wants to use operations that explicitly need a window name as an object, like `moveWindow`. If you call `imshow('test', image);` directly, then it is done internally implicitly. Therefore removing this line completely will not change the functionality of this sample!